src/main/java/frc/robot/Robot.java

```diff
@@ -163,4 +177,10 @@ public void testInit() {
 
     @Override
     public void testPeriodic() {}
+
+    @Override
+    public void simulationInit() {}
+
+    @Override
+    public void simulationPeriodic() {}
```

Comment:
Override `simulationInit()` to keep the same pattern as other modes, override `simulationPeriodic()` to get rid of the default message printed when not overriding it.